pkg/server/plugin/upstreamauthority/awssecret/awssecret_test.go

```diff
@@ -96,13 +100,21 @@ func (as *Suite) Test_MintX509CAInvalidCSR() {
 		csr, _, err := util.NewCSRTemplate(invalidSpiffeID)
 		as.Require().NoError(err)
 
-		resp, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		stream, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		as.Require().NoError(err)
+		as.Require().NotNil(stream)
+
+		resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.